src/style.css

```diff
@@ -1,3 +1,77 @@
+:root {
+  --accent-color: #00acdc;
+  --bg-color: white;
+  --text-color: black;
+  --header-size: 60px;
+}
+
 body {
```

Comment:
The use of the `*` selector is discouraged as it can impact performance. It's better to set styles only for the elements that require them. In this case, you could use `body` instead of `*` to set the `padding` and `margin` to `0`.